### DISCUSSION TITLE
How can I call "replace" without any animations?

### DISCUSSION THREAD

**Discussion by gilbertl** [software-mansion/react-native-screens#771] [Q&A] [answered]
I see that I can customize  stackAnimation and calling navigation.navigate(X) will perform said stackAnimation.

But when I call navigation.replace(X), the stackAnimation property has no effect. I see that there's a property called "replaceAnimation", but I can only specify "push" or "pop"; it cannot be disabled. I also don't recognize any differences between "push" and "pop" on replace.

Any pointers appreciated. Thanks.

**Comment by WoLewicki** [comment, marked as answer]
What do you mean it has no effect? You can e.g. specify `stackAnimation: 'none'` and it should replace the screens without any animation. Am I missing something? Also, could you specify the platform?
